src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -568,6 +641,80 @@ export default class AircraftController {
     }
 
     /**
+     * Generate a unique `transponderCode`
+     *
+     * This method should only be run while building props for a
+     * soon-to-be-instantiated `AircraftModel`
+     *
+     * @for AircraftController
+     * @method _generateUniqueTransponderCode
+     * @return {number}
+     * @private
+     */
+    _generateUniqueTransponderCode() {
+        const minCodeValue = 1203;
+        const maxCodeValue = 7499;
+        const transponderCode = _random(minCodeValue, maxCodeValue);
+
+        if (!this._isValidTransponderCode(transponderCode)) {
+            // the value generated is already in use, recurse back through this method and try again
+            this._generateUniqueTransponderCode();
```

Comment:
You would know. It just seemed to me like it could be dangerous if we start to fill it up (I mean, if we have like 500 planes in the simulation, we'll probably have much bigger issues anyway...) Having to repeat two or three or four times seemed to me like it would be something better avoided. But if it doesn't have an adverse effect on performance, frankly I couldn't care any less! lol